content/python/concepts/tuples/tuples.md

```diff
@@ -15,40 +15,126 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-A **tuple** is a data structure used to store an ordered and immutable collection of objects, unlike [dictionaries](https://www.codecademy.com/resources/docs/python/dictionaries) which are unordered and mutable. They are similar to [lists](https://www.codecademy.com/resources/docs/python/lists), except that tuples cannot be changed after creation.
+A **tuple** is a data structure used to store an ordered and immutable collection of objects, unlike [dictionaries](https://www.codecademy.com/resources/docs/python/dictionaries) which are unordered and mutable. They are similar to [lists](https://www.codecademy.com/resources/docs/python/lists), except that tuples cannot be changed after creation. Tuple elements can be of different [data types](https://www.codecademy.com/resources/docs/python/data-types).
 
-Tuples also support built-in sequence functions such as [`len()`](https://www.codecademy.com/resources/docs/python/built-in-functions/len), [`min()`](https://www.codecademy.com/resources/docs/python/built-in-functions/min), [`max()`](https://www.codecademy.com/resources/docs/python/built-in-functions/max).
+Tuples also support built-in sequence functions such as [`len()`](https://www.codecademy.com/resources/docs/python/built-in-functions/len), [`min()`](https://www.codecademy.com/resources/docs/python/built-in-functions/min), and [`max()`](https://www.codecademy.com/resources/docs/python/built-in-functions/max).
 
 ## Syntax
 
 ```pseudo
 # With built-in function
-tuple(value1, value2, ... valueN)
+tuple(value1, value2, ..., valueN)
 
 # With parentheses (multi-item)
-tuple = (value1, value2, ... valueN)
+tuple = (value1, value2, ..., valueN)
 
 # With parentheses (single item with a trailing comma)
 tuple = (singleValue, )
+
+# With no parentheses (but with a trailing comma)
+tuple = singleValue,
 ```
 
 There are multiple ways to create tuples in Python:
```

Comment:
It seems weird that lines 38-41 come after the syntax examples. I think they'd be better before line 24.

Reply:
I see your point. I think the sentence on line # 38 might make more sense going above the line # 24. But I think I'm going to keep the list items below the \```pseudo snippet and convert into a paragraph on one line (followed by the **Note** that build upon the sentence about creating w/ parentheses, I think).